kedro/io/data_catalog.py

```diff
@@ -681,29 +681,41 @@ def add_all(
             self.add(name, dataset, replace)
 
     def add_feed_dict(self, feed_dict: dict[str, Any], replace: bool = False) -> None:
-        """Adds instances of ``MemoryDataset``, containing the data provided
-        through feed_dict.
+        """This function adds datasets to the ``DataCatalog`` using the data provided through the `feed_dict`.
```

Comment:
I'm still missing a bit more explanation here about what `feed_dict` is. Adding that would make a huge difference to me.

Reply:
Done